composer.json

```diff
@@ -9,13 +9,13 @@
 		}
 	],
 	"require-dev": {
+		"nextcloud/coding-standard": "^1.0",
+		"nextcloud/ocp": "dev-stable26",
```

Comment:
This should always track the lowest supported version I guess?

Reply:
yes

Reply:
Added some CI that will send the update PRs